Plugins/SequencePlugin/Source/SequencePlugin/Public/Marketplace/Structs/SeqGetCollectiblesWithLowestListingsArgs.h

```diff
@@ -0,0 +1,50 @@
+#pragma once 
+
+#include "CoreMinimal.h"
+#include "SeqCollectiblesFilter.h" 
+#include "SeqMarketplacePage.h"
+#include "SeqGetCollectiblesWithLowestListingsArgs.generated.h"
+
+USTRUCT(BlueprintType)
+struct SEQUENCEPLUGIN_API FSeqGetCollectiblesWithLowestListingsArgs
+{
+    GENERATED_USTRUCT_BODY()
+
+public:
+    // Properties
+    UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Collectibles")
+    FString ContractAddress;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Collectibles")
+    FSeqCollectiblesFilter Filter;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Collectibles")
+    FSeqMarketplacePage Page;
+
+    bool customGetter = true;
+
+    FString GetArgs()
+    {
+        FString ret = "{";
+
+        ret += "\"contractAddress\":\"" + ContractAddress + "\"";
+
+        if (Filter.ContainsData())
+        {// Check if the Filter has data and append its args if it does
+            ret.Append(",\"filter\":");
+            ret.Append(Filter.GetArgs());
+        }
+
+        // Check if the Page has data and append its args if it does
+        if (Page.ContainsData())
```

Comment:
Doesn't a Page have default values? If so, won't this always be true? 

The reason I ask is because, I was fixing some of the Indexer behaviour with Pages and I made the FSeqPage a TOptional. I'm wondering if what you've done here is a better approach.
https://github.com/0xsequence/sequence-unreal/pull/150/commits/dc1758ec4385d10e99aa43ff7422a4ba98bde114

What do you think?

Reply:
I tried making Page into a `TOptional` similar to how you used in in your mentioned commit, @BellringerQuinn, and it all still seemed to function fine.  However, lets say you were working with this class through blueprints, if you "Make SeqGetCollectiblesWithLowestListingsArgs" from the base function node, it will now be missing the `page` input value.

May be best to keep it the way @caballoninja had it.